packages/shared/src/features/onboarding/shared/Header.tsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import type { ReactElement } from 'react';
+import classNames from 'classnames';
+import {
+  Button,
+  ButtonSize,
+  ButtonVariant,
+} from '../../../components/buttons/Button';
+import { IconSize } from '../../../components/Icon';
+import { MoveToIcon } from '../../../components/icons';
+import Logo, { LogoPosition } from '../../../components/Logo';
+
+interface ProgressBarProps {
+  currentChapter: number;
+  currentStep: number;
+  chapters: { steps: number }[];
+  className?: string;
+}
+
+function ProgressBar({
```

Comment:
Could probably be it's own component as well.
Was also wondering if we should maybe use new folder structure here for web funnels already?
CC: @ilasw think you had some ideas on structure right?

Reply:
this is already the new structure afaik.
i'll extract the progress bar to be standalone, no problem